content/architecture/index.textile

```diff
@@ -0,0 +1,141 @@
+** Automated routing of all traffic away from an abrupt failure of datacenter in less than two minutes
+* **Availability**: Meticulously designed to provide continuity of service even in the case of instance or whole datacenter failures.
+** 99.999% global service availability (no more than 5 minutes 15 seconds cumulative downtime per year)
+** 50% global capacity margin for instant demand surges
+
+h3. Design Objectives
+
+Ably's platform is a global service that supports all of our realtime messaging and associated services. It is architected to achieve horizontal scalability with "no effective ceiling":https://ably.com/blog/ablys-four-pillars-no-scale-ceiling on application scale, while maintaining consistent latency, message integrity, and system reliability across our global network.
+
+The platform has been designed with the following primary objectives in mind:
+
+* **Horizontal scalability**: As more nodes are added, load is automatically redistributed across the cluster so that global capacity increases linearly with the number of instances we run.
+* **No single point of congestion**: As the system scales, there is no single point of congestion for any data path, and data within the system is routed peer-to-peer, ensuring no single component becomes overloaded as traffic scales for an individual app or across the cluster.
+* **Fault tolerance**: Faults in the system are expected, and the system must have redundancy at every layer in the stack to ensure availability and reliability.
+* **Autonomy**: Each component in the system should be able to operate fully without reliance on a global controller. For example, two isolated data centers should continue to service realtime requests while isolated.
+* **Consistent low latencies**: Within data centers, we aim for latencies to be in the low 10s of milliseconds and less than 100ms globally. Consistently achieving low latencies requires careful consideration of the placement of data and services across the system as well as prioritisation of the computation performed by each service.
+* **Quality of service**: We intentionally design for specific integrity and consistency properties to enable sophisticated realtime applications that would be impossible on platforms with weaker guarantees.
+
+h3. Cluster Architecture
+
+Ably's platform runs on AWS EC2 infrastructure with a globally distributed architecture. Our "clusters":/docs/platform-customization#dedicated-and-isolated-clusters typically span multiple regions, usually between two and ten. This multi-region approach maximizes availability and is a critical aspect of providing a fault-tolerant service.
+
+Each regional deployment operates independently, handling its own subscriber connections, REST traffic, channel management and message routing. When activity occurs on a channel across multiple regions, messages flow peer-to-peer between regions directly, eliminating central bottlenecks and single points of failure.
```

Comment:
More of another point for the future; but I think it would be worth adding a brief overview of connections/channels/messages and the pub/sub pattern at the beginning of this page to provide some context to points such as these.